test/irb/test_init.rb

```diff
@@ -66,6 +66,57 @@ def test_rc_file_in_subdir
       end
     end
 
+    def test_rc_files
+      tmpdir = @tmpdir
+      Dir.chdir(tmpdir) do
+        ENV["XDG_CONFIG_HOME"] = "#{tmpdir}/xdg"
+        IRB.conf[:RC_NAME_GENERATOR] = nil
+        assert_includes IRB.rc_files, tmpdir+"/.irb#{IRB::IRBRC_EXT}"
```

Comment:
Let's just call it `.irbrc`:
```suggestion
        assert_includes IRB.rc_files, tmpdir+"/.irbrc"
```

I think we can change all `IRB::IRBRC_EXT` references in this file to just `rc` as I don't think we'd ever change it.